geotrek/feedback/helpers.py

```diff
@@ -82,9 +82,23 @@ def post_to_suricate(self, endpoint, params):
             )
         else:
             response = requests.post(f"{self.URL}{endpoint}", params)
+        self.check_response_integrity(response)
 
+    def get_attachment_from_suricate(self, url):
+        if self.USE_AUTH:
+            response = requests.get(
+                url,
+                auth=self.AUTH,
+            )
+        else:
+            response = requests.get(
+                url,
+            )
         if response.status_code not in [200, 201]:
```

Comment:
Should we handle [http 304 status code](https://httpwg.org/specs/rfc7232.html#status.304) ?
This status code is often found when there is some sort of caching in place.
If you know that your http client does transparent caching (it converts 304 response into 200 responses) then we don't need to handle it here.